scapy/config.py

```diff
@@ -686,3 +685,17 @@ def func_in(*args, **kwargs):
                               "Please install python-cryptography v1.7 or later.")  # noqa: E501
         return func(*args, **kwargs)
     return func_in
+
+
+def scapy_delete_temp_files():
+    # type: () -> None
+    for f in conf.temp_files:
+        try:
+            os.unlink(f)
+        except Exception:
+            pass
+    del conf.temp_files[:]
+
+
+conf = Conf()
```

Comment:
Is there a reason why you decided to move this here?

Reply:
I moved it back (Done).